osu.Game.Rulesets.Osu/Skinning/LegacyMainCirclePiece.cs

```diff
@@ -80,6 +80,11 @@ Texture getTextureWithFallback(string name)
 
                 return tex ?? skin.GetTexture($"hitcircle{name}");
             }
+
+            bool overlayAboveNumber = skin.GetConfig<OsuSkinConfiguration, bool>(OsuSkinConfiguration.HitCircleOverlayAboveNumber)?.Value ?? true;
```

Comment:
should be above the nested function, probably right beneath the children definition so it's less likely it is missed when other drawables are (potentially) added, causing breaking behaviour.